src/mame/video/undrfire.cpp

```diff
@@ -287,9 +288,9 @@ void undrfire_state::draw_sprites_cbombers(screen_device &screen, bitmap_ind16 &
 			sprite_ptr->zoomx = zx << 12;
 			sprite_ptr->zoomy = zy << 12;
 
-			if (primasks)
+			if (pritable)
 			{
-				sprite_ptr->primask = primasks[priority];
+				sprite_ptr->primask = u32(~1) << u32(pritable[priority]);
```

Comment:
Why do you need to cast the shift up to `u32` here?  It shoudn’t make any difference as the value has the be less than 32 anyway.